WooCommerce/src/androidTest/kotlin/com/woocommerce/android/e2e/screens/mystore/StatsComponent.kt

```diff
@@ -33,6 +33,13 @@ class StatsComponent : Screen(STATS_DASHBOARD) {
         }
     }
 
+    fun switchToStatsDashboardTodayTab(): MyStoreScreen {
+        selectItemWithTitleInTabLayout(R.string.today, R.id.app_bar_layout)
+        waitForGraphToLoad()
+
+        return MyStoreScreen()
+    }
```

Comment:
looks like there's a potential to not repeat the 4 similar functions and combine them into something like:

```suggestion
    fun switchToStatsDashboardTab(duration: String): MyStoreScreen {
        val tab = mapOf(
            "today" to R.string.today,
            "week" to R.string.this_week,
            "month" to R.string.this_month,
            "year" to R.string.this_year
        )

        // there was a type mismatch error and the !! fixed it, not sure if there are better solutions around that
        selectItemWithTitleInTabLayout(tab[duration]!!, R.id.app_bar_layout)
        waitForGraphToLoad()

        return MyStoreScreen()
    }
```

Reply:
Thanks for the idea @jostnes! I did this in 33e67dc7e02105ed805e37fa41a9425ed4fc1991, but a bit differently, to have less hardcoded strings.